test/test_notebooks.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+from pathlib import Path
+import nbformat
+from nbconvert.preprocessors import ExecutePreprocessor, CellExecutionError
+
+
+@pytest.mark.parametrize(
+    ("notebook_filename", "timeout"),
+    [
+        ("Tutorials/COSIMA_CookBook_Tutorial.ipynb", 600),
+        ("Tutorials/Make_Your_Own_Database.ipynb", 600),
```

Comment:
```suggestion
```

Reply:
I've now commented the notebooks that do not work or that have not been tested yet.